close_all_positions.py

```
@@ -0,0 +1,105 @@
+                }
+
+                query_string = "&".join([f"{k}={v}" for k, v in params.items()])
+                signature = hmac.new(
+                    binance['api_secret'].encode('utf-8'),
+                    query_string.encode('utf-8'),
+                    hashlib.sha256
+                ).hexdigest()
+
+                params['signature'] = signature
+
+                response = requests.post(order_url, params=params, headers=headers)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)